### ISSUE TITLE
Fix capitalisation of python (prefer Python) as per Vale suggestion

### ISSUE THREAD

**Issue by stichbury** [mckinsey/vizro#734]
Vale recommends we use "Python" rather than "python".

<img width="620" alt="image" src="https://github.com/user-attachments/assets/cc1e51ce-3dfc-44f8-b466-0e30d60bc06b">

Task: Search across Vizro docs (vizro-core and vizro-ai) for instances of python (not Python) and replace with discretion. 

If there are genuine cases where lower case `python` is more appropriate, use the following to switch Vale off/on in situ

```markdown
<!-- vale off -->
Legitimate use of lower case p ("python")
<!--vale on-->
```

**Comment by feridaaa** [comment]
Hi! I would like to work on this issue.

**Comment by antonymilne** [comment]
Hello @feridaaa, did you make any progress on this? Are you still interested on working on it? 🙂 No pressure if not!

**Comment by antonymilne** [comment]
Closed by #794 🎉 Thank you @feridaaa!
